tests/functional/disaster-recovery/regional-dr/test_acm_kubevirt_dr_integration.py

```
@@ -331,33 +359,965 @@ def test_acm_kubevirt_using_different_protection_types(
+            wait_for_resource_existence(
+                kind=constants.DRPC,
+                namespace=constants.DR_OPS_NAMESPACE,
+                resource_name=resource_name,
+                timeout=120,
+                should_exist=False,
+            )
+            logger.info("Standalone VM DR protection removed and DRPC deleted")
+
+    # TODO: Add Polarion ID when available
+    @pytest.mark.polarion_id("OCS-zzzz")
+    def test_acm_kubevirt_mixed_protection_types(
```

Ruff (0.15.20)

[warning] Too many branches (17 > 12)

(PLR0912)